pkgs/development/python-modules/pudb/default.nix

```diff
@@ -3,27 +3,30 @@
 , fetchPypi
 , pygments
 , urwid
+, urwid-readline
+, jedi
 , isPy3k
 }:
 
 buildPythonPackage rec {
   pname = "pudb";
-  version = "2020.1";
+  version = "2021.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "2ef23d3371439f13fffbe7f8b9fc180a19d4183dc9cab204d878d7c7766464bf";
+    sha256 = "sha256-MJ7oK0Wg/8oLxMf1If0+NXWJx2TzOb353Ku3rUBpLW4=";
   };
 
-  propagatedBuildInputs = [ pygments urwid ];
+  propagatedBuildInputs = [ pygments urwid urwid-readline jedi ];
 
   # Tests fail on python 3 due to writes to the read-only home directory
   doCheck = !isPy3k;
 
+  pythonImportsCheck = [ "pudb" ];
+
   meta = with lib; {
     description = "A full-screen, console-based Python debugger";
     license = licenses.mit;
     platforms = platforms.all;
```

Comment:
You want to add yourself as a maintainer?

```suggestion
```
defaults to the platforms of python.